core-web/libs/sdk/editor/project.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "sdk-editor",
```

Comment:
Same here

Reply:
Here is correct. This is the name of the project of Nx.